src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -983,6 +1017,10 @@ def _ensure_default_log_analytics_workspace_for_monitoring(cmd, subscription_id,
 
 
 def _ensure_container_insights_for_monitoring(cmd, addon):
+    # Workaround for this addon key which has been seen lowercased in the wild.
```

Comment:
nit: Can you please keep this comment lowercase to start and no period at the end to stay consistent